deserializers/src/main/java/com/nextdoor/bender/deserializer/json/GenericJsonDeserializerConfig.java

```diff
@@ -32,6 +33,16 @@
   @JsonProperty(required = false)
   private List<FieldConfig> nestedFieldConfigs = Collections.emptyList();
 
+  @JsonSchemaDescription("When true the deserializer will assume that the raw event strings are base64 encoded and will"
+          + "attempt to decode them and then unzip.")
+  @JsonSchemaDefault(value = "false")
+  @JsonProperty(required = false)
+  private Boolean performBase64DecodeAndUnzip = false;
```

Comment:
Nit on naming here, we should be saying `gzip` instead of `unzip` since they're different. And same thing for the description of `bufferSize`

Reply:
done